### PR TITLE
support header

### PR DESCRIPTION
**Pull request by shawncx**
Support header in response and parameter

### CODE REVIEW

File: powershell/llcsharp/schema/string.ts [hunk 1]
```diff
@@ -398,10 +398,10 @@ ${this.validateEnum(eventListener, property)}
     return `await ${eventListener}.AssertRegEx(${nameof(property.value)},${property},@"${pattern}");`;
   }
   private validateEnum(eventListener: Variable, property: Variable): string {
-    if (!(this.schema instanceof ChoiceSchema)) {
+    if (this.schema.type !== SchemaType.Choice && this.schema.type !== SchemaType.SealedChoice) {
```
Comment (dolauli) [inline powershell/llcsharp/schema/string.ts:401]: should just this.schema.type !== SchemaType.SealedChoice

File: powershell/llcsharp/schema/string.ts [hunk 2]
```diff
@@ -398,10 +398,10 @@ ${this.validateEnum(eventListener, property)}
...
       return '';
     }
-    const choiceValues = this.schema.choices.map((c) => c.value);
+    const choiceValues = (<ChoiceSchema | SealedChoiceSchema>this.schema).choices.map((c) => c.value);
```
Comment (dolauli) [inline powershell/llcsharp/schema/string.ts:404]: SealedChoiceSchema only

File: powershell/llcsharp/schema/string.ts (hunk 1 above)
Comment (shawncx) [reply powershell/llcsharp/schema/string.ts]: Got it. Will fix in next PR

**Pull request merged**